src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/RegexIgnoreCaseTests.cs

```diff
@@ -90,6 +90,7 @@ public async Task IgnoreCase_Behavior_Is_Constant(RegexEngine engine)
         }
 
         [Theory]
+        [SkipOnPlatform(TestPlatforms.Browser, "Browser uses Invariant mode which causes invariant mappings to be used and will fail to match '\u0130' and 'I'")]
```

Comment:
We need to skip the whole test, or just certain inputs?

And would this need to be skipped if RegexOptions.InvariantCulture were specified?

Will this test fail as written if run on a machine set to have a tr-TR culture by default?

Reply:
RegexOptions.InvariantCulture will never be specified since I'm manually only passing in options as RegexOptions.IgnoreCase, but you do have a point where the test will fail as written if culture is invariant or one of the 8 turkish ones. Also, to your first question, the only input that fails is the '\u0130' and 'I' mapping, so I can just skip that one in browser case. I will fix the test with all of the proposed changes.